proprietary/tokens/src/brand/ams/border.tokens.json

```diff
@@ -0,0 +1,10 @@
+{
+  "ams": {
+    "border-width": {
```

Comment:
Considering these are the ‘border’ tokens, as the file name implies, it would make sense to split `border` and `width` here. This allows us to add the likes of `ams.border.style` later as siblings below `border`.

The custom properties in CSS will still match the property name. In JSON and JS, ‘width’ and ‘style’ will become children of ‘border’ which is conceptually also correct.